### PR TITLE
org & workspace

### PR DESCRIPTION
**Pull request by captain-Akshay**
**Notes for Reviewers**

This PR fixes #

[Screencast from 2024-04-29 18-47-19.webm](https://github.com/meshery/meshery/assets/59491379/e1f2863b-785c-4769-a477-1c7432b69c13)

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by dragon-slayer875 — CHANGES_REQUESTED** [review]
Please check these out.

**Comment by leecalcote** [comment]
merge conflicts...

**Comment by leecalcote** [comment]
@captain-Akshay merge conflicts...

File: ui/assets/icons/WorkspaceOutlinedIcon.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+function WorkspaceOutlinedIcon({ width = '24px', height = '24px', fill = '#F6F8F8' }) {
```
Comment (ShivamCoder23) [inline ui/assets/icons/WorkspaceOutlinedIcon.js:3]: put theme in sistent

**Review by ShivamCoder23 — CHANGES_REQUESTED** [review]

**Comment by leecalcote** [comment]
@captain-Akshay merge conflicts...

**Pull request closed without merging**